grants/czi-eoss5-notebook-v7-loi.md

```diff
@@ -0,0 +1,75 @@
+## Proposal Title: Auto-filled; Maximum of 60 characters, including spaces.
+
+Jupyter Notebook maintenance and community support for v4 migration.
+
+## Amount Requested: Total budget amount requested in USD, including indirect costs; this number should be between $100,000 USD and $400,000 USD total costs over a two-year period.
+
+$400,000 USD
+
+## Proposal Summary/Scope of Work: Provide a short summary of the work being proposed (maximum of 500 words).
+
+JupyterLab and the Classic Notebook are the two major user interfaces to manipulate the Jupyter notebook documents. That format is really popular within the scientific community thanks to its ability to bring code, data analysis and visualization in a single document.
+
+This proposal follows the major the upcoming major version of the Classical Notebook will be built on top of JupyterLab plugins (that use newer web technologies). The three key ideas approved in the related migration enhancement proposal are: document-centric user experience, extensions are critical to the community and existing educational content relies on the current Notebook experience. The user interface changes try to be limited (although new features will require adaptation). But the extension API will be totally different.
+
+As maintenance is mainly done by a couple of volunteers during their free time. The backlog of issues is ever growing due to the lack of manpower. Addressing that, supporting extensions migration and the new user issues for the new version will require an important effort.
```

Comment:
```suggestion
The backlog of issues is growing because maintenance is done by only a couple of volunteers during their spare time. It is important to address this backlog by supporting extensions migration and by anticipating new user issues for the new software versions.
```